pydis_site/apps/api/migrations/0095_user_display_name.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.0.3 on 2024-04-01 12:21
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("api", "0094_migrate_mailing_listdata"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="user",
+            name="display_name",
+            field=models.CharField(blank=True, help_text="The display name, taken from Discord.", max_length=32),
+        ),
+        migrations.RunSQL("UPDATE api_user SET display_name = name;"),
```

Comment:
Please pass `elidable=True`, as we don't need this migration when squashing (I'm planning to squash migrations soon, see also #1280)
```suggestion
        migrations.RunSQL("UPDATE api_user SET display_name = name;", elidable=True),
```